network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreRepository.java

```diff
@@ -1169,6 +1177,13 @@ public void deleteNetwork(UUID uuid) {
         batch.add(delete().from(HVDC_LINE).where(eq("networkUuid", uuid)));
         batch.add(delete().from(DANGLING_LINE).where(eq("networkUuid", uuid)));
         session.execute(batch);
+
+        try {
```

Comment:
Do not make a rest request from repository.
The reports deletion must be made in the study server

Reply:
done